sqlmesh/core/snapshot/definition.py

```diff
@@ -683,7 +650,16 @@ def missing_intervals(
             )
             compare_ts = seq_get(dates, i + lookback) or dates[-1]
 
-            for low, high in self.intervals:
+            #  in missing intervals check if is and forward only and paused and if so use dev intervals
```

Comment:
>  if is and forward only and paused

not sure I fully understand the sentence

Reply:
that was a note to myself. thanks for catching.